CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Knot.x Releases
+All notable changes to Knot.x will be documented in this file.
+
+## Unreleased
+List of changes that are finished but not yet released in any final version.
+
+## Version 1.0.0 (*In progress*)
+- Initial open source release
+
+## Version [1.0.0-RC5](https://github.com/Cognifide/knotx/releases/tag/1.0.0-RC5)
+- Changed Knot.x modules naming convention to io.knotx.*.
+- [fixed Form submission](https://github.com/Cognifide/knotx/issues/237).
```

Comment:
I think, better is to use Pull request ID as it effectively provides the changes, so you can later easily see what was changed in the code. Maybe use following syntax:
```
- [PR 123](http://link_to_PR)
```